### PR TITLE
Switching from yarn dependency to npm

### PR DESCRIPTION
**Pull request by jonrohan**
We want to be more inclusive for contributors, so we're switching to using `npm` as the package manager tool of choice. This is more inclusive because npm is included with nodejs, so there's less a potential contributor might need to install on their machine.

### CODE REVIEW

File: .github/workflows/docs-preview-deploy.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
           key: gems-deploy-preview-${{ hashFiles('**/Gemfile.lock') }}
 
       - name: Install dependencies
-        run: yarn && cd docs && yarn && cd ..
+        run: npm i && cd docs && yarn && cd ..
```
Comment (neall) [inline .github/workflows/docs-preview-deploy.yml:41]: Did you mean to leave "docs" using yarn?

Comment (jonrohan) [reply .github/workflows/docs-preview-deploy.yml:41]: Was looking at it now and running into dependency issues, not sure if you have suggestions?

```
@jonrohan ➜ /workspaces/view_components/docs (remove_yarn) $ npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @primer/react@35.2.2
npm ERR! Found: react@16.13.1
npm ERR! node_modules/react
npm ERR!   react@"^16.13.1" from the root project
npm ERR!   peer react@"^0.14.0 || ^15.0.0 || ^16.0.0" from create-react-context@0.2.2
npm ERR!   node_modules/create-react-context
npm ERR!     create-react-context@"0.2.2" from react-live@2.2.2
npm ERR!     node_modules/react-live
npm ERR!       react-live@"^2.1.2" from @primer/gatsby-theme-doctocat@4.1.0
npm ERR!       node_modules/@primer/gatsby-theme-doctocat
npm ERR!         @primer/gatsby-theme-doctocat@"^4.1.0" from the root project
npm ERR!   65 more (downshift, formik, framer-motion, gatsby, gatsby-cli, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17.0.0" from @primer/react@35.2.2
npm ERR! node_modules/@primer/react
npm ERR!   @primer/react@"^35.2.2" from the root project
npm ERR!   @primer/react@"^35.2.2" from @primer/gatsby-theme-doctocat@4.1.0
npm ERR!   node_modules/@primer/gatsby-theme-doctocat
npm ERR!     @primer/gatsby-theme-doctocat@"^4.1.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: react@17.0.2
npm ERR! node_modules/react
npm ERR!   peer react@"^17.0.0" from @primer/react@35.2.2
npm ERR!   node_modules/@primer/react
npm ERR!     @primer/react@"^35.2.2" from the root project
npm ERR!     @primer/react@"^35.2.2" from @primer/gatsby-theme-doctocat@4.1.0
npm ERR!     node_modules/@primer/gatsby-theme-doctocat
npm ERR!       @primer/gatsby-theme-doctocat@"^4.1.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/vscode/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vscode/.npm/_logs/2022-09-20T15_59_59_131Z-debug-0.log`
```

Comment (keithamus) [reply .github/workflows/docs-preview-deploy.yml:41]: Make an `.npmrc` file with the contents `legacy-peer-deps=true`.

**Comment by jonrohan** [comment]
This is currently blocked until Kuby supports a npm package https://github.com/getkuby/kuby-core/tree/master/lib/kuby/docker/packages

**Review by camertron — APPROVED** [review]

**Pull request merged**